test/functional/tool_utils.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+# Copyright 2014 BitPay Inc.
+# Copyright 2016-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit.
+"""Exercise the utils via json-defined tests."""
+
+from test_framework.test_framework import BitcoinTestFramework
+
+import difflib
+import json
+import logging
```

Comment:
In faa18bf287fc02339b7af29d54af862a289bf96d "test: Turn util/test_runner into functional test"

It shouldn't be necessary to import logging as each previous `logging.` call can be converted into a `self.log.`